sherpa/stats/__init__.py

```diff
@@ -147,7 +148,8 @@ def __init__(self, name='cash'):
         Likelihood.__init__(self, name)
 
     @staticmethod
-    def calc_stat(data, model, staterror=None, syserror=None, weight=None):
+    def calc_stat(data, model, staterror=None, syserror=None, weight=None, \
+                      bkg=None):
```

Comment:
Should we verify that bkg is None here (to catch the case when a user is calling something incorrectly)? For instance, add

    if bkg is None:
        raise ValueError("The background can not be included in the statistic")

but with an improved error message?

Or is this limiting, and we may actually get cases where bkg is set here but its safe to ignore?

This holds for all the remaining stats which have a bkg parameter but do not use it.